send/pithia_portal

```diff
@@ -1,4 +1,42 @@
-#!/bin/bash
-export SERVICE_NAME="pithia_portal"
+#!/usr/bin/env python3
+import os
+import sys
+import subprocess
+import send_lib
+import requests
+import generic_sender
 
-python3 generic_send.py "$1" "$2" "$3"
+SERVICE_NAME = "pithia_portal"
+os.environ["SERVICE_NAME"] = SERVICE_NAME
```

Comment:
I don't think we need to export service name as environmental variable anymore as it is passed directly to methods it is used in, and its not used in methods missing it in their API.